FSharp.Core.Nuget/FSharp.Core.nuspec

```diff
@@ -114,5 +114,9 @@
     <file src="..\packages\Microsoft.FSharp.Core.netcore.1.0.0-rc-170122\lib\netstandard1.6\FSharp.Core.optdata" target="lib\netstandard1.6\FSharp.Core.optdata" />
     <file src="..\packages\Microsoft.FSharp.Core.netcore.1.0.0-rc-170122\lib\netstandard1.6\FSharp.Core.sigdata" target="lib\netstandard1.6\FSharp.Core.sigdata" />
     <file src="..\packages\Microsoft.FSharp.Core.netcore.1.0.0-rc-170122\lib\netstandard1.6\FSharp.Core.xml" target="lib\netstandard1.6\FSharp.Core.xml" />
+    <file src="..\packages\Microsoft.FSharp.Core.netcore.1.0.0-rc-170122\lib\netstandard1.6\FSharp.Core.optdata" target="runtimes\any\native\FSharp.Core.optdata" />
```

Comment:
I'm not familiar with ``runtimes\any`` and ``runtimes\native`` - why are only the sigdata/optdata files there, and not the DLLs?  Normally we expect those files to always only be alongside FSharp.Core.dll?